docker/data/logstash/env2yaml/env2yaml.go

```diff
@@ -126,8 +126,6 @@ var validSettings = []string{
 	"xpack.management.elasticsearch.ssl.cipher_suites",
 	"xpack.geoip.download.endpoint",
 	"xpack.geoip.downloader.enabled",
-	"cloud.id",
```

Comment:
Are these "cloud" settings related to the module being removed?

Reply:
I see these are removed in i18n etc too.

Reply:
Is `cloud.id` here _only_ relevant to modules, or is this where it is also set for central management and internal collection?

Reply:
cloud.id is only used for modules
CPM uses `xpack.management.elasticsearch.cloud_id`. Legacy monitoring uses `xpack.monitoring.elasticsearch.cloud_id`